src/app/tests/suites/certification/Test_TC_MC_3_5.yaml

```diff
@@ -28,10 +28,10 @@ tests:
               - name: "nodeId"
                 value: nodeId
 
-    # TODO: Need to enable when we can properly verify vendor id and uint16
-    - label: "Read launch list attribute."
-      disabled: true
+    #Issue 1078
```

Comment:
This can't be the right issue number.

Reply:
We need the right value that need to be checked in the List

Reply:
Yes, but issue 1078 has nothing to do with vendor ids.